antenna/configlib.py

```diff
@@ -91,6 +101,22 @@ def parse_bool(val):
     raise ValueError('%s is not a valid bool value' % val)
 
 
+def parse_class(val):
+    """Parses a string, imports the module and returns the class
+
+    >>> parse_class('hashlib.md5')
+
+    """
+    module, class_name = val.rsplit('.', 1)
+    module = importlib.import_module(module)
+    try:
+        return getattr(module, class_name)
+    except AttributeError:
+        raise ValueError('%s is not a valid member of %s' % (
```

Comment:
Continuing, I think the actual exception is moot. This is something that the developer is seeing when they're implementing something and a user is seeing if they've got antenna configured wrong. It's not an exception that is intended to be caught and handled.

I'm on the fence. I think I want to hedge towards users being able to see the message and intuit what to do next. I don't know what to do to make it better.

With Python 3, this is much easier since you can chain exceptions. Then we could throw an additional `ConfigurationError` with a super helpful message and the user gets our message plus the original one.